dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-http/src/main/java/org/apache/dolphinscheduler/plugin/alert/http/HttpSender.java

```diff
@@ -108,8 +113,15 @@ public AlertResult send(String msg) {
     }
 
     public String getResponseString(HttpRequestBase httpRequest) throws IOException {
-        CloseableHttpClient httpClient =
-                HttpClients.custom().setRetryHandler(HttpServiceRetryStrategy.retryStrategy).build();
+        RequestConfig requestConfig = RequestConfig.custom()
```

Comment:
Use TimeUnit to convert s to ms or directly use timeout * 1000 is ok, no need to define 
NUMBER_1000

Reply:
Done